Hints.py

```diff
@@ -368,6 +368,13 @@ def at(spot):
 
         raise HintAreaNotFound('No hint area could be found for %s [World %d]' % (spot, spot.world.id))
 
+    @classmethod
+    def get_dungeon_color(cls, dungeon_name):
```

Comment:
I'd suggest replacing this with a `for_dungeon` class method and calling `.color` on the resulting object. You should also prefer `hint_area.dungeon_name` over `hint_area.value[4]` since the positions of the columns can change (e.g. #1641 adds a column that would break this code).

Reply:
I seem to remember having an issue using `.dungeon_name` and `.color` at some point while iterating over them, but whatever it is no longer seems to apply so those will be changed. I plan on making a few more changes to this process so I'll take the rest under consideration when I work on it more.